hif/pcie/8964/tx_ndp.c

```diff
@@ -334,13 +334,8 @@ int pcie_tx_init_ndp(struct ieee80211_hw *hw)
 	struct ieee80211_tx_info *tx_info = IEEE80211_SKB_CB(&skb);
 	int rc;
 
-	if (sizeof(struct pcie_tx_ctrl_ndp) >
-	    sizeof(tx_info->driver_data)) {
-		wiphy_err(hw->wiphy, "driver data is not enough: %zu (%zu)\n",
-			  sizeof(struct pcie_tx_ctrl_ndp),
-			  sizeof(tx_info->driver_data));
-		return -ENOMEM;
-	}
+	BUILD_BUG_ON(sizeof(struct pcie_tx_ctrl_ndp) >
```

Comment:
Would it make sense to use BUILD_BUG_ON_MSG instead?

Reply:
+1 for @neheb

Reply:
Yes, thank you for maintaining this message, it's an initialization function, it doesn't cost anything in terms of performance.